packages/components/statistic/src/index.vue

```diff
@@ -0,0 +1,208 @@
+<!-- eslint-disable @typescript-eslint/no-this-alias -->
```

Comment:
Need to cancel the eslint rule.

Reply:
Ok, I will adjust it as soon as possible